biolink-model.yaml

```diff
@@ -2725,21 +2950,103 @@ classes:
 #    mixin: true
 
   data set:
-    is_a: named thing
-    mappings:
+    is_a: information content entity
+    exact_mappings:
       - IAO:0000100
+      - dctypes:Dataset
 
   data set version:
     is_a: data set
     slots:
-      - title
-#      - description
       - source data file
       - version of
       - type
       - distribution
-    mappings:
-      - dctypes:Dataset
+
+  # TODO: to review additional ontology relating to Publications, such as http://www.sparontologies.net/ontologies
+  publication:
+    is_a: information content entity
+    description: >-
+      Any published piece of information. Can refer to a whole publication,
```

Comment:
See my comment above about the scope of 'publication' . . . here, what is a 'whole publication' when we are talking about scientific journal content.  Is the publication a single article? a single issue?  the journal as a whole (all issues/volumes)?  The current definition seems to propose that 'publication' covers all of these concepts?

Consider if it might we useful to define the concept of a single 'published work' -  a single discrete work by a particular author (or set of authors).  e.g. each article in a journal, not the journal itself; or each essay in a published collection, not the collection or larger periodical.  If 'publication' is meant to be broader than this - perhaps a separate class for 'published work'.

Reply:
@mbrush please see my extensive (LOL) reply above (next to your comment above)